modules/protocols/snap/include/virgil/iot/protocols/snap/prvs/prvs-client.h

```diff
@@ -88,27 +90,33 @@ extern "C" {
  * \a stop_wait_func member or #vs_snap_prvs_client_impl_t structure.
  * \a wait_func member or #vs_snap_prvs_client_impl_t structure.
  *
+ * \param[in] ctx Service context.
  * \param[in] condition Condition buffer. Must not be NULL.
  * \param[in] expect Expected value to be set.
  *
  * \return #VS_CODE_OK in case of success or error code.
  */
-typedef vs_status_e (*vs_snap_prvs_stop_wait_t)(int *condition, int expect);
+typedef vs_status_e (*vs_snap_prvs_stop_wait_t)(struct vs_snap_prvs_client_impl_t *ctx, int *condition, int expect);
```

Comment:
I disagree with parameter names like client_mpl because we need to pass the same self pointer in all other modules. I know that  I am  late but we need to decrease the entropy influence

Reply:
Also, pay attention to `struct`